packages/web3-providers-http/types/tests/web3-provider-http-tests.ts

```diff
@@ -40,15 +39,13 @@ const options: HttpProviderOptions = {
         http: new http.Agent({}),
         https: new https.Agent({})
     }
-};
-
-const httpProvider = new HttpProvider('http://localhost:8545', options);
+});
 
 // $ExpectType void
-httpProvider.send(payload, (error: Error | null) => {});
+httpProvider.send({} as any, (error: Error | null) => {});
```

Comment:
This test is weaker than before and I think those type tests should be as strict as possible.

Reply:
I've resolved this and created a related issue for it (https://github.com/ethereum/web3.js/issues/3335)